xml/System.Security/CodeAccessPermission.xml

```diff
@@ -971,8 +971,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Throws a <see cref="T:System.Security.SecurityException" /> at run time if the security requirement is not met.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.CodeAccessPermission> instance is cast to an <xref:System.Security.IPermission> interface.
+
+          ]]></format>
+        </remarks>
```

Comment:
```suggestion
        <summary>To be added.</summary>
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.CodeAccessPermission> instance is cast to an <xref:System.Security.IPermission> interface.

          ]]></format>
        </remarks>
        <exception cref="T:System.Security.SecurityException">The security requirement is not met.</exception>
```